pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -23,23 +23,44 @@ import (
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// AzVolume is a specification for a AzVolume resource
+// AzVolume is a specification for an AzVolume resource
 type AzVolume struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   AzVolumeSpec   `json:"spec"`
-	Status AzVolumeStatus `json:"status"`
+	// spec defines the desired state of an AzVolume.
+	// Required.
+	Spec AzVolumeSpec `json:"spec"`
+	// status represents the current state of AzVolume.
+	// Nil status indicates that the underlying volume has not yet been provisioned
+	// +optional
+	Status *AzVolumeStatus `json:"status,omitempty"`
 }
 
-// AzVolumeSpec is the spec for a AzVolume resource
+// AzVolumeSpec is the spec for an AzVolume resource
 type AzVolumeSpec struct {
 	UnderlyingVolume string `json:"underlyingVolume"`
+
+	MaxMountReplicaCount int `json:"maxMountReplicaCount"`
+	//The suggested name for the storage space
```

Comment:
Is this different from the name of the `AzVolume` object (`<AzVolume>.ObjectMeta.Name`)?

Reply:
No, I'll remove the extra field